src/components/Header.js

```diff
@@ -0,0 +1,20 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import '../styles/header.css';
+
+const Header = () => (
+  <div>
+    <div className="header">
+      <h1>Math Magicians</h1>
+      <div className="ulDiv">
+        <ul>
+          <li><Link to="/">Home</Link></li>
+          <li><Link to="/calculator">Calculator</Link></li>
+          <li><Link to="/quote">Quotes</Link></li>
```

Comment:
- Optional
I would strongly recommend using `NavLink` component instead of using `Link`. You can read the difference between the two [here](https://www.educative.io/answers/what-are-the-differences-between-navlink-link-and-a-in-react).